kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ReplaceQuestions.vue

```diff
@@ -208,17 +209,9 @@
         context.emit('closePanel');
       }
 
-      const replacements = ref([]);
-
-      function handleReplacement() {
+      function submitReplacement() {
+        handleReplacement();
```

Comment:
Is there any particular reason to call this from within a function rather than returning `handleReplacement` from the setup function?

Reply:
I believe we needed the router logic to be placed inside the `ReplaceQuestions` file itself. Returning `handleReplacement` from the setup function and directly calling it from `@submit` handler would require the routing logic to be placed in the `handleReplacement  function` that is now moved to `useQuizCreation`.
That was the main reason for me to call it from within a function as that provides separation of concerns.

Though I'd like to know if I'm missing something here and if there's a better workaround for this.

Reply:
Oh 😅. No problem :)